_episodes/24-diagnosing-issues-improving-robustness.md

```diff
@@ -425,9 +425,9 @@ def patient_normalise(data):
 
     Negative values are rounded to 0.
     """
-    max = np.nanmax(data, axis=1)
+    maxima = np.nanmax(data, axis=1)
```

Comment:
I think we do want to call this variable `max` as we want the linter to thrown the error about "Redefining built-in 'max'" later on.

Reply:
Yes, you are correct. I'll remove this change.

Reply:
```suggestion
    max = np.nanmax(data, axis=1)
```